docs/Doxyfile_common

```diff
@@ -755,42 +755,48 @@ WARN_LOGFILE           =
 # Note: If this tag is empty the current directory is searched.
 
 INPUT                  = . \
-                         ../src/nupic/encoders/ScalarEncoder.hpp \
-                         ../src/nupic/encoders/ScalarSensor.hpp \
-                         ../src/nupic/algorithms/SpatialPooler.hpp \
-                         ../src/nupic/algorithms/TemporalMemory.hpp \
-                         ../src/nupic/algorithms/Connections.hpp \
-                         ../src/nupic/algorithms/SDRClassifier.hpp \
-                         ../src/nupic/algorithms/ClassifierResult.hpp \
-                         ../src/nupic/engine/NuPIC.hpp \
-                         ../src/nupic/engine/Network.hpp \
-                         ../src/nupic/engine/Region.hpp \
-                         ../src/nupic/engine/RegionImpl.hpp \
-                         ../src/nupic/engine/RegisteredRegionImpl.hpp \
-                         ../src/nupic/engine/RegionImplFactory.hpp \
-                         ../src/nupic/engine/Input.hpp \
-                         ../src/nupic/engine/Output.hpp \
-                         ../src/nupic/engine/Link.hpp \
-                         ../src/nupic/engine/LinkPolicy.hpp \
-                         ../src/nupic/engine/LinkPolicyFactory.hpp \
-                         ../src/nupic/engine/UniformLinkPolicy.hpp \
-                         ../src/nupic/ntypes/Dimensions.hpp \
-                         ../src/nupic/types/BasicType.hpp \
-                         ../src/nupic/types/Fraction.hpp \
-                         ../src/nupic/types/Serializable.hpp \
-                         ../src/nupic/math/Domain.hpp \
-                         ../src/nupic/math/Index.hpp \
-                         ../src/nupic/math/NearestNeighbor.hpp \
-                         ../src/nupic/math/SegmentMatrixAdapter.hpp \
-                         ../src/nupic/math/Set.hpp \
-                         ../src/nupic/math/SparseMatrixAlgorithms.hpp \
-                         ../src/nupic/math/SparseMatrixConnections.hpp \
-                         ../src/nupic/math/SparseRLEMatrix.hpp \
-                         ../src/nupic/math/SparseTensor.hpp \
-                         ../src/nupic/math/Topology.hpp \
-                         ../src/nupic/engine/Link.hpp \
-                         ../src/nupic/engine/Output.hpp \
-                         ../src/nupic/engine/Input.hpp
+                         ../src/htm/encoders \
```

Comment:
ok as-is, but isn't there some regex or something to look for all necessary files within a path (`../src/`) ?